src/models/2014/equipment/types.d.ts

```diff
@@ -54,7 +54,7 @@ export type Equipment = {
   _id?: mongoose.Types.ObjectId;
   armor_category?: string;
   armor_class?: ArmorClass;
-  capacity?: number;
+  capacity?: string;
```

Comment:
Did this type change for 2014?

Reply:
It has actually always been a string, the type was just wrong.